intl/msg_hash_us.h

```diff
@@ -87,7 +87,7 @@ MSG_HASH(
    MENU_ENUM_LABEL_VALUE_DUMP_DISC,
    "Dump Disc"
    )
-MSG_HASH(
+MSG_HASH( /* FIXME It is not immediately clear what "saved as an image file" means. Is it a specific image type? Determined automatically? User's choice? */
```

Comment:
Seems like the Crowdin system doesn't take too fondly the usage of quotation marks on a comment. Can you remove them? Also, disc-based media, like CD-ROMs or DVDs, are usually preserved into what's called an image file, as in, an image of the disc. See https://en.wikipedia.org/wiki/Disk_image

Reply:
I know what an image file is, but I also know that there are several formats as well. I'm merely trying to say that it probably should be mentioned which image format will be used (ISO, BIN & CUE, something else, determined automatically, set by user, etc). I mean, this might no longer be relevant, but "back in my day" some emulators were quite picky about the image format.

Reply:
Can you push a PR that fixes these things for Discord? It will be important to do this as soon as possible so localizers are not held up by this.